lib/mapreduce/mapreduce.go

```diff
@@ -527,3 +528,13 @@ func WaitJob(outRepo, branch, commit, job string) error {
 	log.Printf("WaitJob: %s %s %s %s", outRepo, branch, commit, job)
 	return btrfs.WaitForFile(path.Join(outRepo, branch, ".progress", commit, job))
 }
+
+func getDockerHost() (host string) {
+	host = Os.Getenv("DOCKER_HOST")
```

Comment:
Hey,

For the record, I think `DOCKER_HOST` is actually the better option here. `DOCKER_HOST`, `DOCKER_TLS_VERIFY`, and `DOCKER_CERT_PATH` are environment variables not only recognized natively by a docker client, but other docker-based processes (including docker-compose). My problem here would be the lack of differentiating between docker daemons running with TLS and those which do not, which also needs to be handled.

(Bad, ie do not use this) example: https://github.com/peter-edge/go-dockerutils/blob/master/dockerutils.go#L55